src/pages/About/AboutKyberSwap/MeetTheTeam.tsx

```diff
@@ -0,0 +1,247 @@
+import { Trans } from '@lingui/macro'
+import { Flex, Text } from 'rebass'
+import styled from 'styled-components'
+
+import ImranMohamad from 'assets/images/kyber_members/imran_mohamad.png'
+import Jane from 'assets/images/kyber_members/jane.png'
+import LoiLuu from 'assets/images/kyber_members/loi_luu.png'
+import MikeLe from 'assets/images/kyber_members/mike_le.png'
+import NguyenKimTrong from 'assets/images/kyber_members/nguyen_kim_trong.png'
+import PrakharAgarwal from 'assets/images/kyber_members/prakhar_agarwal.png'
+import SashaMai from 'assets/images/kyber_members/sasha_mai.png'
+import ShaneHong from 'assets/images/kyber_members/shane_hong.png'
+import SpyrosVrettos from 'assets/images/kyber_members/spyros_vrettos.png'
+import TuNguyen from 'assets/images/kyber_members/tu_nguyen.png'
+import VictorTran from 'assets/images/kyber_members/victor_tran.png'
+import { ReactComponent as LinkedInIcon } from 'assets/svg/linkedin.svg'
+import { ReactComponent as TwitterIcon } from 'assets/svg/solid_twitter_icon.svg'
+import useTheme from 'hooks/useTheme'
+
+type Member = {
+  name: string
+  title: string
+  avatarUrl: string
+
+  handles: {
```

Comment:
Why named it `handles`. Is `link` better?

Reply:
Use `handles` because the value is a handle, such as `vutran54`, not the full url https://twitter.com/vutran54